docs/content/configuration/broker.md

```diff
@@ -67,7 +67,6 @@ The broker uses processing configs for nested groupBy queries. And, optionally,
 
 |Property|Description|Default|
 |--------|-----------|-------|
-|`druid.query.groupBy.singleThreaded`|Run single threaded group By queries.|false|
```

Comment:
i have just removed it from the broker docs because it does not seem to be doing anything at the broker. i think, it only applies to historicals.
default is still totally overridableby setting druid.query.groupBy.singleThreaded property at historicals and historical doc already has it.